src/main/java/com/es/phoneshop/model/product/service/CartServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.es.phoneshop.model.product.service;
+
+import com.es.phoneshop.model.product.bean.Cart;
+import com.es.phoneshop.model.product.bean.CartItem;
+import com.es.phoneshop.model.product.bean.Product;
+import com.es.phoneshop.model.product.dao.ArrayListProductDao;
+
+import javax.servlet.http.HttpSession;
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.util.List;
+
+public class CartServiceImpl implements CartService {
+    private static volatile CartService instance;
+    private final static String NOT_NUMBER = "Not a number";
+    private final static String OUT_OF_STOCK = "Out of stock, available ";
```

Comment:
Please align your modifiers order with jls - https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.3.1 
Check and fix for the entire project